ts/components/services/ServicesOther.tsx

```diff
@@ -1,6 +1,7 @@
+// TODO: delete this component (bieng substituted with the generic ServicesSectionsList)
```

Comment:
same here. You could append this comment to the following one

Reply:
file removed (https://github.com/teamdigitale/io-app/commit/4a5010d53af5dcbf687bc50cc6ec1cff542f3554)